internal/clusters_test.go

```diff
@@ -209,15 +209,21 @@ func TestAccClustersApiIntegration(t *testing.T) {
 	require.NoError(t, err)
 	assert.True(t, len(events) > 0)
 
+	// List clusters in workspace
 	all, err := w.Clusters.ListAll(ctx, compute.ListClustersRequest{})
 	require.NoError(t, err)
+	foundCluster := false
+	for _, info := range all {
+		if info.ClusterName == clusterName {
+			foundCluster = true
+		}
+	}
+	assert.True(t, foundCluster)
```

Comment:
Maybe we could use: https://pkg.go.dev/golang.org/x/exp/slices#Contains and if not then let's make a generic utility so it can be used at other places: https://stackoverflow.com/a/70802740/16597007.